test/jdk/javax/swing/JOptionPane/bug4174551.java

```diff
@@ -23,7 +23,7 @@
 
 /*
  * @test
- * @bug 4174551
+ * @bug 4174551 8346260
```

Comment:
Should the test exercise all available Look and Feels?

Reply:
SInce it's manual test, I guess this will make it more complex..its good as it is..

Reply:
Fair enough.

Reply:
Does the year for this test have to be updated as well? Curious since it's only adding a bugID, but we're still touching the test.

Reply:
We are not changing any test logic so I am refraining from changing copyright year..